packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -940,6 +940,29 @@ export default class RestApi {
         },
       },
     }
+    collections.groups.actions = {
+      update: withParams(
+        async ({ id, name, userIds }, req) => {
+          const group = await app.getGroup(id)
+          if (group.provider !== undefined) {
+            throw new Error('Cannot edit synchronized group')
+          }
+
+          if (name !== undefined) {
+            await app.updateGroup(id, { name })
+          }
+
+          if (Array.isArray(userIds)) {
+            await app.setGroupUsers(id, userIds)
+          }
+        },
+        {
+          id: { type: 'string' },
+          name: { type: 'string', optional: true },
+          userIds: { type: 'array', items: { type: 'string' }, optional: true },
+        }
+      ),
+    }
```

Comment:
Don't create action like that. Same as: https://github.com/vatesfr/xen-orchestra/pull/8276#discussion_r1933602634
```suggestion
```